src/__tests__/components/fields/chips/chips.spec.tsx

```diff
@@ -0,0 +1,146 @@
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+import { IChipsSchema } from "../../../../components/fields";
+import { FrontendEngine, IFrontendEngineData } from "../../../../components/frontend-engine";
+import { TestHelper } from "../../../../utils";
+import { ERROR_MESSAGE, SUBMIT_BUTTON_ID, TOverrideField, TOverrideSchema } from "../../../common";
+
+const submitFn = jest.fn();
+const componentId = "field";
+const fieldType = "chips";
+const componentTestId = TestHelper.generateId(componentId, fieldType);
+
+const textareaLabel = "Durian";
+const textareaId = `chips-${textareaLabel}`;
+const textareaTestId = TestHelper.generateId(textareaId, "textarea");
+
+const renderComponent = (overrideField?: TOverrideField<IChipsSchema>, overrideSchema?: TOverrideSchema) => {
+	const json: IFrontendEngineData = {
+		id: "test",
+		fields: {
+			[componentId]: {
+				label: "Chips",
+				fieldType,
+				options: [
+					{ label: "A", value: "A" },
```

Comment:
can use a label that is different from value, to ensure the respective strings are displayed and submitted